src/Entities/Boat.h

```diff
@@ -30,14 +42,15 @@ class cBoat :
 	virtual void Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk) override;
 	virtual void HandleSpeedFromAttachee(float a_Forward, float a_Sideways) override;
 
-	cBoat(double a_X, double a_Y, double a_Z);
+	/** The type is the wood type of the boat. There are five boat types (0 = oak, 1 = spruce, 2 = birch, 3 = jungle, 4 = acacia, 5 = dark oak) */
```

Comment:
... not to mention that the comment is inaccurate: there are 6 types; it's the maximum type value that is 5.